components/Discussion/DiscussionProvider/components/DiscussionMetaHelper.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { inQuotes } from '@project-r/styleguide'
+
+import { useDiscussion } from '../context/DiscussionContext'
+import Meta from '../../../Frame/Meta'
+import { getFocusUrl } from '../../CommentLink'
+import { useTranslation } from '../../../../lib/withT'
+
+/**
+ * Render meta tags for a focused comment.
+ * @constructor
+ */
+const DiscussionMetaHelper = () => {
+  const { t } = useTranslation()
+  const { discussion, loading, error } = useDiscussion()
+
+  if (loading || error || !discussion) {
+    return null
+  }
+
+  if (discussion.comments.focus) {
```

Comment:
should also depend on `router.query.focus` IMO to avoid intermediate state issue

Reply:
I don't see how that would help. Since there is a good chance that the comment to focus hasn't been loaded yet, I see no benefit im executing this twice.

Reply:
Also as of now we never update the focusId while the page has already loaded.

Reply:
ok, i was more thinking to match the behaviour/check in article page but in the end this probably won't matter. If I find a bug I'll report it again but otherwise fine with leaving as is